### PR TITLE
[IMP] iot_oca: Add tags, groups and new menus

### PR DESCRIPTION
**Pull request by etobella**
@lubusax @pedrobaeza

### CODE REVIEW

File: iot_oca/models/iot_device_group.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IotDeviceGroup(models.Model):
+    _name = "iot.device.group"
+    _description = "Iot Group"
+
+    name = fields.Char()
```
Comment (pedrobaeza) [inline iot_oca/models/iot_device_group.py]: required=True

File: iot_oca/models/iot_device_tag.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import fields, models
+
+
+class IotDeviceTag(models.Model):
+
+    _name = "iot.device.tag"
+    _description = "Device Tag"
+
+    name = fields.Char()
```
Comment (pedrobaeza) [inline iot_oca/models/iot_device_tag.py]: required=True

File: iot_oca/views/iot_device_group.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        <field name="model">iot.device.group</field>
+        <field name="arch" type="xml">
+            <form>
+                <header/>
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                    </group>
+                </sheet>
+                <div class="oe_chatter"/>
```
Comment (pedrobaeza) [inline iot_oca/views/iot_device_group.xml]: This can go out, as it doesn't inherit `mail.thread`

File: iot_oca/views/iot_device_group.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                <field name="name"/>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="iot_device_group_tree_view">
+        <field name="name">iot.device.group.tree (in iot_oca)</field>
+        <field name="model">iot.device.group</field>
+        <field name="arch" type="xml">
+            <tree>
```
Comment (pedrobaeza) [inline iot_oca/views/iot_device_group.xml]:
```suggestion
            <tree editable="bottom">
```

File: iot_oca/views/iot_device_tag.xml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                <header/>
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                    </group>
+                    <group name="main">
+                        <field name="color" />
+                    </group>
+                </sheet>
+                <div class="oe_chatter"/>
```
Comment (pedrobaeza) [inline iot_oca/views/iot_device_tag.xml]: This can go out, as it doesn't inherit `mail.thread`

File: iot_oca/views/iot_device_tag.xml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="name"/>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="iot_device_tag_tree_view">
+        <field name="name">iot.device.tag.tree (in iot_oca)</field>
+        <field name="model">iot.device.tag</field>
+        <field name="arch" type="xml">
+            <tree>
```
Comment (pedrobaeza) [inline iot_oca/views/iot_device_tag.xml]:
```suggestion
            <tree editable="bottom">
```

File: iot_oca/views/iot_device_tag.xml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="name"/>
```
Comment (pedrobaeza) [inline iot_oca/views/iot_device_tag.xml]:
```suggestion
                <field name="name"/>
                <field name="color"/>
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please check Travis

**Review by albariera — APPROVED** [review]

**Pull request merged**